src/core/server.py

```
@@ -224,29 +290,108 @@ async def crawl4ai_lifespan(server: FastMCP) -> AsyncIterator[Crawl4AIContext]:
+        # Basic health status
+        health_status = {
+            "status": "healthy",
+            "timestamp": time.time(),
+            "version": "1.0.0"
+        }
+        
+        # Add server metrics
+        try:
+            server_metrics = monitor.get_server_metrics()
+            health_status["metrics"] = server_metrics
+        except Exception as metrics_error:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)